tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -542,18 +549,33 @@ user verifies release summary
     user checks summary list contains    Release type    ${RELEASE_TYPE}
 
 user changes methodology status to Approved
+    [Arguments]
+    ...    ${publishing_strategy}=Immediately
+    ...    ${with_release}=
+    ${is_publishing_strategy_with_release}=    Evaluate    '${publishing_strategy}' == 'WithRelease'
     user clicks button    Edit status
     user clicks element    id:methodologyStatusForm-status-Approved
     user enters text into element    id:methodologyStatusForm-latestInternalReleaseNote    Approved by UI tests
+    user clicks element    id:methodologyStatusForm-publishingStrategy-${publishing_strategy}
+    IF    '${publishing_strategy}' == 'WithRelease'
```

Comment:
Or can write `IF    ${is_publishing_strategy_with_release} is ${TRUE}`.  I'm thinking that was the original intention here :)

Reply:
Ah that was the original intention, thanks. I just forgot to use it. 🙂